dataprofiler/labelers/data_processing.py

```diff
@@ -62,18 +82,22 @@ def __eq__(self, other):
         :return: Whether or not self and other are equal
         :rtype: bool
         """
-        if type(self) != type(other) or self._parameters != other._parameters:
+        if (
+            type(self) != type(other)
+            or not isinstance(other, BaseDataProcessor)
```

Comment:
same, another mypy confused?

Reply:
`type(self) != type(other)` should validate that `not isinstance(other, BaseDataProcessor)` would return `True`. Just a redundant check